sessionize/app/src/main/java/co/touchlab/sessionize/MainApp.kt

```diff
@@ -64,7 +64,9 @@ class MainApp :Application(){
         AppContext.initAppContext ()
 
         AppContext.dataLoad()
```

Comment:
I think dataload shouldn't have to wait on the users choice of notifications, so I don't think it should be inside the notification initialization. The reason I had it in dataload is in case the session information changed, so I think it should stay in there as well.

Reply:
You did make me realize that the call in dataLoad should be in dataCalls instead, because that's when the sessions are actually updating.

Reply:
I'm not sure what you mean, It looks like dataCalls is called from dataLoad. so I'm not sure how you would reverse that.

What I was thinking is that we could remove AppContext line 105 because we haven't initialized yet and are about to